Documentation/UpdatingToGA.md

```diff
@@ -16,8 +44,13 @@ In some instances, multiple assemblies have been merged to create better unity o
 
 The following tables describe how the RC2 .asmdef file names map to the GA release. All assembly names match the .asmdef file name. 
 
+- [MixedRealityToolkit](#mixedrealitytoolkit)
+- [MixeddRealityToolkit.Providers](#mixedrealitytoolkit.providers)
```

Comment:
Misspelled: MixeddRealityToolkit -> MixedRealityToolkit

Here and below.

Reply:
removing these as the doc tool apparently is not fond of bookmarks with a '.' in them. (this was the failure in doc generation: "illegal link")